src/device.rs

```diff
@@ -68,9 +68,10 @@ impl Device {
 
         let flags = OFlag::O_RDWR
             | OFlag::O_CLOEXEC
-            | match config.non_blocking_dqbuf {
-                true => OFlag::O_NONBLOCK,
-                false => OFlag::empty(),
+            | if config.non_blocking_dqbuf {
+                OFlag::O_NONBLOCK
+            } else {
+                OFlag::empty()
```

Comment:
Interesting the clippy would complain about this, in effect this adds one line of code. :)